Typelib does not build

I get the following while building current master of typelib:

```
[  2%] Building CXX object typelib/CMakeFiles/typeLib.dir/value_ops.o
/home/sempromuser/RockDev/tools/typelib/typelib/value_ops.cc: In function ‘boost::tuples::tuple<unsigned char*, __gnu_cxx::__normal_iterator<const unsigned int*, std::vector<unsigned int> > > Typelib::ValueOps::zero(uint8_t*, std::vector<unsigned int>::const_iterator, std::vector<unsigned int>::const_iterator)’:
/home/sempromuser/RockDev/tools/typelib/typelib/value_ops.cc:304:39: error: ‘memset’ was not declared in this scope
/home/sempromuser/RockDev/tools/typelib/typelib/value_ops.cc: In function ‘boost::tuples::tuple<unsigned char*, unsigned char*, __gnu_cxx::__normal_iterator<const unsigned int*, std::vector<unsigned int> > > Typelib::ValueOps::copy(uint8_t*, uint8_t*, std::vector<unsigned int>::const_iterator, std::vector<unsigned int>::const_iterator)’:
/home/sempromuser/RockDev/tools/typelib/typelib/value_ops.cc:450:51: error: ‘memcpy’ was not declared in this scope
/home/sempromuser/RockDev/tools/typelib/typelib/value_ops.cc: In function ‘boost::tuples::tuple<bool, unsigned char*, unsigned char*, __gnu_cxx::__normal_iterator<const unsigned int*, std::vector<unsigned int> > > Typelib::ValueOps::compare(uint8_t*, uint8_t*, std::vector<unsigned int>::const_iterator, std::vector<unsigned int>::const_iterator)’:
/home/sempromuser/RockDev/tools/typelib/typelib/value_ops.cc:507:55: error: ‘memcmp’ was not declared in this scope
/home/sempromuser/RockDev/tools/typelib/typelib/value_ops.cc: In member function ‘virtual void VectorOutputStream::write(const uint8_t*, size_t)’:
/home/sempromuser/RockDev/tools/typelib/typelib/value_ops.cc:582:46: error: ‘memcpy’ was not declared in this scope
/home/sempromuser/RockDev/tools/typelib/typelib/value_ops.cc: In member function ‘virtual void ByteArrayOutputStream::write(const uint8_t*, size_t)’:
/home/sempromuser/RockDev/tools/typelib/typelib/value_ops.cc:736:44: error: ‘memcpy’ was not declared in this scope
/home/sempromuser/RockDev/tools/typelib/typelib/value_ops.cc: In member function ‘virtual void VectorInputStream::read(uint8_t*, size_t)’:
/home/sempromuser/RockDev/tools/typelib/typelib/value_ops.cc:845:55: error: ‘memcpy’ was not declared in this scope
/home/sempromuser/RockDev/tools/typelib/typelib/value_ops.cc: In member function ‘virtual void ByteArrayInputStream::read(uint8_t*, size_t)’:
/home/sempromuser/RockDev/tools/typelib/typelib/value_ops.cc:899:55: error: ‘memcpy’ was not declared in this scope
/home/sempromuser/RockDev/tools/typelib/typelib/value_ops.cc: In member function ‘virtual void FDOutputStream::write(const uint8_t*, size_t)’:
/home/sempromuser/RockDev/tools/typelib/typelib/value_ops.cc:643:32: warning: ignoring return value of ‘ssize_t write(int, const void*, size_t)’, declared with attribute warn_unused_result [-Wunused-result]
make[2]: *** [typelib/CMakeFiles/typeLib.dir/value_ops.o] Error 1
make[1]: *** [typelib/CMakeFiles/typeLib.dir/all] Error 2
make: *** [all] Error 2
```

Resettign to 3d598531e227d63f7571df4a38bacc6ed723ced5 works as workaround.

--- a/typelib/value_ops.cc
+++ b/typelib/value_ops.cc
@@ -5,10 +5,9 @@
 #include <iostream>
 #include "value_ops_details.hh"
 #include <cstdio>
+#include <cstring>
 
 using namespace Typelib;
-using std::memcmp;
-using std::memcpy;
 
 void Typelib::display(std::ostream& io, MemoryLayout::const_iterator const begin, MemoryLayout::const_iterator const end)
 {

